src/object/srv_obj_migrate.c

```diff
@@ -1292,9 +1310,6 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 	int i;
 	int rc = 0;
 
-	if (obj_shard_is_ec_parity(mrone->mo_oid, mrone->mo_dkey_hash, &mrone->mo_oca))
-		return migrate_fetch_update_parity(mrone, oh, ds_cont);
-
```

Comment:
DASSERT(obj_shard_is_ec_date(mrone->mo_oid, mrone->mo_dkey_hash, &mrone->mo_oca))?

Reply:
Not sure what you mean here? these are removed lines?

Reply:
just mean seems the assertion should be true here.

Reply:
ah, ok